src/test/scala/com/advancedtelematic/director/http/DeviceResourceSpec.scala

```diff
@@ -454,8 +474,8 @@ trait DeviceResourceSpec extends DirectorSpec with KeyGenerators with DefaultPat
     // currently device-current-target and device-update-target are both at 2
     // sending empty device manifest should not update device-current-target to 3
     updateManifestOk(device, deviceManifest2)
-    deviceVersion(device) shouldBe Some(3)
-    deviceScheduledVersion(device) shouldBe 3
+    deviceVersion(device) shouldBe Some(2)
```

Comment:
this should be written: `should contain 2` so that the matcher gives a nicer msg if it fails

Reply:
Ah cool!